File: BDH.Rhino.Web.API.Domain/Solvers/School/Models/GeneratedClusterCellResponse.cs
using BDH.Rhino.Web.API.Domain.Solvers.School.Models;

namespace BDH.Rhino.Web.API.Domain.Models
{
    public class GeneratedClusterCellResponse
    {
        public GeneratedPointResponse Point { get; set; } = null!;
        public int Hoogte { get; set; }
        public bool[] NorthFacades { get; set; } = Array.Empty<bool>();
        public bool[] EastFacades { get; set; } = Array.Empty<bool>();
        public bool[] SouthFacades { get; set; } = Array.Empty<bool>();
        public bool[] WestFacades { get; set; } = Array.Empty<bool>();
    }
}

File: BDH.Rhino.Web.FrontEnd/src/app/modules/edu-solve/models/DataTransferObjects/GeneratedClusterCellResponse.ts
import { GeneratedPointResponse } from './GeneratedPointResponse';


export interface GeneratedClusterCellResponse {
    point: GeneratedPointResponse;
    hoogte: number;
    northFacades: boolean[];
    eastFacades: boolean[];
    southFacades: boolean[];
    westFacades: boolean[];
}
